src/faq.pug

```diff
@@ -6,16 +6,16 @@ block content
             :markdown-it
                 # Frequently Asked Questions
                 Here you will find the answer to the most common issues first-time users of Stryker encounter.  
-                If this page does not answer your question, please feel free to [![Gitter](https://badges.gitter.im/stryker-mutator/stryker.svg)](https://gitter.im/stryker-mutator/stryker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge).
+                If this page doesn't answer your question, please feel free to [![Gitter](https://badges.gitter.im/stryker-mutator/stryker.svg)](https://gitter.im/stryker-mutator/stryker?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge).
                 If you think you've encountered a bug, please also contact us on Gitter first, before submitting a bug report.
 
                 ## Q: Stryker is slow! How do I speed it up?  
-                **A: Improve your tests' performance**  
```

Comment:
I think the two spaces at the end here tell markdown that the next line should be on a newline. So it is needed (but I can't check atm)

Reply:
I have made a dot in the first commit, I haven't noticed that I removed space in the second commit